### PR TITLE
Gravity document testing

### PR DESCRIPTION
**Pull request by EntyR**

### CODE REVIEW

File: app/src/test/java/com/harman/roomdbapp/BottomSheetViewModelTest.kt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    @BeforeEach
+    fun setUp() {
+        useCase = mockk(relaxed = true)
+    }
+
+    @Test
+    fun `Verify gravity document entry's are received`() {
+
+        val fileName = "123456.csv"
+        val init = listOf(GravityRecord(1f, 1L))
```
Comment (AlexeyKolosov) [inline app/src/test/java/com/harman/roomdbapp/BottomSheetViewModelTest.kt]: lets don't use kotlin reserved keywords like "init"

File: app/src/test/java/com/harman/roomdbapp/DataStorageViewModelTest.kt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    @BeforeEach
+    fun setUp() {
+        useCase = mockk(relaxed = true)
+    }
+
+    @Test
+    fun `Verify gravity document list is received`() {
+
+        val init = listOf("initValue")
```
Comment (AlexeyKolosov) [inline app/src/test/java/com/harman/roomdbapp/DataStorageViewModelTest.kt]: the same here

Comment (EntyR) [reply app/src/test/java/com/harman/roomdbapp/DataStorageViewModelTest.kt]: done

File: app/src/test/java/com/harman/roomdbapp/BottomSheetViewModelTest.kt (hunk 1 above)
Comment (EntyR) [reply app/src/test/java/com/harman/roomdbapp/BottomSheetViewModelTest.kt]: done

File: data/src/test/java/com/harman/roomdbapp/data/repository/DocumentRepositoryTest.kt [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        val initName = "1234.csv"
+        val repository = DocumentRepository(context)
+        repository.deleteCsvFile(initName)
+        Truth.assertThat(result).isTrue()
+    }
+
+    @Test
+    fun `verify last record are saved`() {
+
+        val init = 1234L
```
Comment (AlexeyKolosov) [inline data/src/test/java/com/harman/roomdbapp/data/repository/DocumentRepositoryTest.kt]: init

File: data/src/test/java/com/harman/roomdbapp/data/repository/DocumentRepositoryTest.kt [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+            context.getSharedPreferences(any(), any()).edit().apply()
+        } returns mockk(relaxed = true)
+        val repository = DocumentRepository(context)
+        repository.saveLastRecord(init)
+        io.mockk.verify { context.getSharedPreferences(any(), any()).edit().putLong(any(), init) }
+    }
+
+    @Test
+    fun `verify last is received`() {
+        val init = 1234L
```
Comment (AlexeyKolosov) [inline data/src/test/java/com/harman/roomdbapp/data/repository/DocumentRepositoryTest.kt]: init

File: data/src/test/java/com/harman/roomdbapp/data/repository/DocumentRepositoryTest.kt [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+        every { context.getSharedPreferences(any(), any()).getLong(any(), any()) } returns init
+        val repository = DocumentRepository(context)
+        Truth.assertThat(repository.getLastRecord()).isEqualTo(init)
+    }
+
+    @Test
+    fun `verify convert is right`() {
+        val init = GravityRecord(1f, 1)
```
Comment (AlexeyKolosov) [inline data/src/test/java/com/harman/roomdbapp/data/repository/DocumentRepositoryTest.kt]: init

File: domain/src/test/java/com/harman/roomdbapp/domain/use_cases/GravityDocumentsUseCaseTest.kt [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+    @BeforeEach
+    fun setUp() {
+        repository = mockk(relaxed = true)
+    }
+
+    @Test
+    fun `Verify gravity document are adding to repository`() {
+
+        val fileName = "1234.csv"
+        val init = listOf(GravityRecord(1f, 1234))
```
Comment (AlexeyKolosov) [inline domain/src/test/java/com/harman/roomdbapp/domain/use_cases/GravityDocumentsUseCaseTest.kt]: init

File: domain/src/test/java/com/harman/roomdbapp/domain/use_cases/GravityDocumentsUseCaseTest.kt [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+        )
+        every { repository.getCsvList() } returns numbers
+        val useCase = GravityDocumentsUseCase(repository)
+        Truth.assertThat(useCase.getGravityDocumentsName()).isEqualTo(numbers)
+    }
+
+    @Test
+    fun `Verify last record is saved`() {
+
+        val init = 16L
```
Comment (AlexeyKolosov) [inline domain/src/test/java/com/harman/roomdbapp/domain/use_cases/GravityDocumentsUseCaseTest.kt]: init

File: domain/src/test/java/com/harman/roomdbapp/domain/use_cases/GravityDocumentsUseCaseTest.kt [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+            result = init
+        }
+        val useCase = GravityDocumentsUseCase(repository)
+        useCase.setLastBackupRecordTimestamp(init)
+        Truth.assertThat(result).isEqualTo(result)
+    }
+    @Test
+    fun `Verify last record is received`() {
+
+        val init = 16L
```
Comment (AlexeyKolosov) [inline domain/src/test/java/com/harman/roomdbapp/domain/use_cases/GravityDocumentsUseCaseTest.kt]: init

File: app/src/test/java/com/harman/roomdbapp/DataStorageViewModelTest.kt (hunk 2 above)
Comment (AlexeyKolosov) [reply app/src/test/java/com/harman/roomdbapp/DataStorageViewModelTest.kt]: Are you sure, that you removed all "init"?

Comment (EntyR) [reply app/src/test/java/com/harman/roomdbapp/DataStorageViewModelTest.kt]: wow, didn't expect to have so many of them, now all renamed

**Review by AlexeyKolosov — APPROVED** [review]

**Pull request merged**